Resources/Prototypes/Reagents/chemicals.yml

```
@@ -80,3 +80,21 @@
         conditions:
         - !type:ReagentThreshold
           min: 5
+
+- type: reagent
+  id: SodiumPolyacrylate
+  name: reagent-name-sodium-polyacrylate
+  desc: reagent-desc-sodium-polyacrylate
+  flavor: bitter
+  physicalDesc: reagent-physical-desc-grainy
+  color: "#F0F0F0"
+  metabolisms:
+    Poison:
+      effects:
+      - !type:SatiateThirst
+        factor: -5 # This is basically industrial dessicant
+      - !type:PopupMessage
+        visualType: MediumCaution
+        type: Local
+        messages: generic-reagent-effect-parched
```

```suggestion
        messages: [ "generic-reagent-effect-parched" ]
```

This needs to be an array to fix the yaml linter error

---

Ah so that's what I need to do. Thanks, I'll do that. I was actually waiting on help in #howdoicode on the discord.